scripts/reset.js

```diff
@@ -0,0 +1,26 @@
+const ADB = require('appium-adb');
+const B = require('bluebird');
+const { flatMap } = require('lodash');
+
+const SERVER_PKGS = [
+  'io.appium.uiautomator2.server',
+  'io.appium.uiautomator2.server.test',
+  'io.appium.settings',
+];
+
+async function runReset () {
+  const adb = new ADB();
+  const udids = (await adb.getConnectedDevices())
+    .filter(({state}) => state === 'device')
+    .map(({udid}) => udid);
+  if (0 === udids.length) {
+    return;
+  }
+  const uninstallPromises = udids.map((udid) => {
```

Comment:
maybe we should add a log line like `Running reset on device ${udid}``

Reply:
Added